README.md

```diff
@@ -1,14 +1,35 @@
 # HyP3-lib
 
-## Common library for HyP3 plugins
+## CURRENTLY UNDER CONSTRUCTION!!!
+HyP3-lib is currently undergoing a major redesign and many utilities that are no longer used by ASF have been removed between version 2 and version 3. In addition, we plan to migrate some remaining functionality to other projects and to significantly refactor what remains. For this reason, **if you are using HyP3-lib we strongly recommend pinning to major versions at this time**. We further recommend not using this library in production until its version 4 release.
 
-### Please note that the use of `hyp3lib` version 2 or less is **_NOT_** recommended. 
+The use of HyP3-lib version 2 or less is **_NOT_** recommended.
 
-This library is outdated and largely untested, containing a broad collection of scripts/tools developed for upstream packages that have since undergone many *major* releases. 
-
-HyP3 plugins only use a subset of this repository, and version 3 will include drastic changes to reduce the library's scope to *just* those codes supporting HyP3 plugin development. We suggest *not* using this library, *especially* in a production environment, until version 3.0. 
+## HyP3-lib: A Common library for HyP3 plugins
```

Comment:
This should be line 1 of the README at a top level header (just `#`)... I've no idea why it was split into two lines before.